Entities/SoundContainer.cpp

```diff
@@ -61,7 +61,9 @@ namespace RTE {
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 	int SoundContainer::ReadProperty(const std::string_view &propName, Reader &reader) {
-		if (propName == "AddSound") {
+		if (propName == "TopLevelSoundSet") {
```

Comment:
Serialization/deserialization should always be reversible. That's my issue - I'd prefer that it is an undocumented field instead, so that anything that is saved can be loaded back in.